components/src/dynamo/vllm/tests/test_vllm_instrumented_scheduler.py

```
@@ -401,6 +467,248 @@ def test_dp_rank_prefers_data_parallel_index():
+        endpoint=endpoint,
+    )
+    coordinator_point = BenchmarkPoint(point_type="decode", benchmark_id=1)
+    follower_point = BenchmarkPoint(
+        point_type="decode", benchmark_id=1, total_kv_read_tokens=16
+    )
+    follower_error = {}
+
+    def synchronize_follower():
+        try:
+            rank1.synchronize(follower_point)
+        except Exception as error:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)